alpa/global_env.py

```diff
@@ -53,6 +53,9 @@ def __init__(self):
         # Cross mesh resharding mode. Possible choices: {"send_recv",
         # "broadcast"}
         self.resharding_mode = "send_recv"
+        # Which nccl to use. Possible choices: {"from_cupy",
+        # "from_xla_extension"}
+        self.nccl_mode = "from_xla_extension"  # "from_cupy"
```

Comment:
I wanna more details about the perf of the xla_extension version because you use block host to replace the default stream.

Reply:
I have changed back to default stream. Please refer to [#tensorflow-alpa PR 121](https://github.com/alpa-projects/tensorflow-alpa/pull/121). This PR is waiting to be reviewed. Thanks!